packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.test.tsx

```diff
@@ -59,19 +65,10 @@ describe('CallWithChatComposite', () => {
 
     const mockCallWithChatAdapter = createMockCallWithChatAdapter();
 
-    const callWithChatComposite = Enzyme.mount(
+    const { container } = render(
       <CallWithChatComposite adapter={mockCallWithChatAdapter} {...mockBaseCompositeProps} />
     );
 
-    const callComposites = callWithChatComposite.find(CallComposite);
-    expect(callComposites.length).toBe(1);
-
-    const callComposite = callComposites.first();
-    expect(callComposite.props().fluentTheme).toBe(mockBaseCompositeProps.fluentTheme);
-    expect(callComposite.props().icons).toBe(mockBaseCompositeProps.icons);
-    expect(callComposite.props().locale).toBe(mockBaseCompositeProps.locale);
-    expect(callComposite.props().rtl).toBe(mockBaseCompositeProps.rtl);
-    expect(callComposite.props().onFetchAvatarPersonaData).toBe(mockBaseCompositeProps.onFetchAvatarPersonaData);
-    expect(callComposite.props().onFetchParticipantMenuItems).toBe(mockBaseCompositeProps.onFetchParticipantMenuItems);
+    expect(container.querySelector('[data-ui-id="lobby-page"]')).toBeTruthy();
```

Comment:
this is the only test I changed _what_ was tested. Before the test simply checked that the props we passed in were the props the composite received -- this doesn't make sense for a test, so I updated this to validate the composite successfully rendered a lobby page as the mock adapter depicts